packages/components/src/components/radio-group/radio-group.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { ChangeEvent } from 'react';
+import classNames from 'classnames';
+import Text from '../text';
+
+type TItem = React.HTMLAttributes<HTMLDivElement> & { value: string; label: string; disabled: boolean };
+type TItemWrapper = {
+    should_wrap_items?: boolean;
+};
+type TRadioGroup = {
+    className?: string;
+    name: string;
+    onToggle: (e: ChangeEvent<HTMLInputElement>) => void;
+    required?: boolean;
+    selected: string;
+} & TItemWrapper;
+
+const ItemWrapper = ({ children, should_wrap_items }: React.PropsWithChildren<TItemWrapper>) => {
+    if (should_wrap_items) {
+        return <div className='dc-radio-group__item-wrapper'>{children}</div>;
+    }
+
+    return <>{children}</>;
+};
+
+const RadioGroup = ({
+    className,
+    name,
+    onToggle,
+    required,
+    selected,
+    should_wrap_items,
+    children,
+}: React.PropsWithChildren<TRadioGroup>) => {
+    const [selected_option, setSelectedOption] = React.useState(selected);
+
+    const onChange = (e: ChangeEvent<HTMLInputElement>) => {
+        setSelectedOption(e.target.value);
+        onToggle(e);
+    };
+
+    return (
+        <div className={classNames('dc-radio-group', className)}>
+            {Array.isArray(children) &&
+                children.map((item, idx) => (
+                    <ItemWrapper key={idx} should_wrap_items={should_wrap_items}>
```

Comment:
Help to change `key={idx}` to another value other than index in your card as well.

Reply:
will replace it to value, as id is not used everywhere